CONFIGURATION.md

```diff
@@ -0,0 +1,128 @@
+# Configuring NTPD-rs
+
+NTPD-rs is an implementation of NTP completely written in Rust, with a focus on exposing a minimal attack surface. The project is currently in an early stage, but if you want to try out NTPD-rs on a non-critical system, this guide provides the basic information needed to properly build and configure NTPD-rs.
+
+## Limitations
+
+The current implementation has several important limitations which it is important to be aware of. First of all, the current implementation is an NTP client only, and does not support acting as an NTP server.
+
+Second, there are currently a few limitations around configuring connections to other NTP nodes:
+ - Only Client-Server connections are supported
```

Comment:
I don't think capitalizing "Client" and "Server" makes sense here. Maybe elaborate on when/where this is an issue? I think most Linux machines probably don't expose the server by default (does chrony?).

Reply:
fixed capitalization.

The limited support is not really an issue for most usecases I know of, however the ntp standard specifies 2 other types of peerings that can exist (broadcast client/server and symmetric active/passive) and people who use ntpd or chronies support for any of those modes should know they are not available.

Reply:
So I would maybe turn this around to prioritize and explicitly name the unsupported scenarios (since this section is focused on limitations), something like:

- No support for broadcast client/server or symmetric active/passive connections, only supports acting as the client towards a server node

Reply:
adopted suggested wording.